include/bluetooth/services/cts_client.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef BT_CTS_CLIENT_H_
+#define BT_CTS_CLIENT_H_
+
+/** @file
+ *
+ * @defgroup bt_gatt_cts_c Current Time Service Client
+ * @{
+ *
+ * @brief Current Time Service Client module.
+ */
+
+#include <bluetooth/gatt.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt_dm.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**@brief "Exact Time 256" field of the Current Time characteristic. */
+struct bt_cts_exact_time_256 {
+	uint16_t year;
+	uint8_t month;
+	uint8_t day;
+	uint8_t hours;
+	uint8_t minutes;
+	uint8_t seconds;
+	uint8_t day_of_week;
+	uint8_t fractions256;
+};
+
+/**@brief "Adjust Reason" field of the Current Time characteristic. */
+struct bt_cts_adjust_reason {
+	uint8_t manual_time_update : 1;
+	uint8_t external_reference_time_update : 1;
+	uint8_t change_of_time_zone : 1;
+	uint8_t change_of_daylight_savings_time : 1;
+};
+
+/**@brief Data structure for the Current Time characteristic. */
+struct bt_cts_current_time_char {
```

Comment:
I would drop the `_char` part. `bt_cts_current_time` is good.

Reply:
Will do.